client/ayon_core/hosts/nuke/api/plugin.py

```diff
@@ -910,8 +903,8 @@ def generate_mov(self, farm=False, **kwargs):
                 self._connect_to_above_nodes(
                     node, product_name, "Reposition node...   `{}`"
                 )
-            # append reformated tag
-            add_tags.append("reformated")
+            # append reformatted tag
+            add_tags.append("reformatted")
```

Comment:
I'm pretty sure we should not be changing these "tags" keys unless we're entirely sure that ALL other code relying on `reformated` to be refactored as well?

E.g. here: https://github.com/ynput/ayon-core/blob/464fc4c78314c2bb3c4999c97337e21118054589/client/ayon_core/plugins/publish/extract_review.py#L1228 and the use of the word `reformated` occurs in many other places instead of just here.